tests/test_zuora_all_fields.py

```diff
@@ -45,7 +45,7 @@ def run_test(self, api_type):
         self.zuora_api_type = api_type
 
         # Streams to verify all fields tests
-        expected_streams = {"RefundTransactionLog", "Account"}
```

Comment:
Skip `AllFields` test for `RefundTransactionLog ` stream due to insufficient data